src/dot-merlin/dot_merlin_reader.ml

```diff
@@ -425,6 +431,10 @@ let postprocess cfg =
         (dirs :> Merlin_dot_protocol.directive list)
       )
     ; (cfg.pass_forward :> Merlin_dot_protocol.directive list)
+    ; cfg.unit_name |> Option.map ~f:(fun name -> `UNIT_NAME name) |> Option.to_list
+    ; cfg.wrapping_prefix
+        |> Option.map ~f:(fun prefix -> `WRAPPING_PREFIX prefix)
+        |> Option.to_list
```

Comment:
Actually, since these variant are part of the `no_processing_required` ones we should simply forward them:
We should remove both `unit_name` and `wrapping_prefix` from `Dot_merlin_reader.config` and simply push them to the `pass_forward` list.